x-pack/solutions/observability/plugins/slo/server/plugin.ts

```diff
@@ -180,6 +189,8 @@ export class SLOPlugin
       ?.start(plugins.taskManager, internalSoClient, internalEsClient)
       .catch(() => {});
 
+    this.tempSummaryCleanupTask?.start({ taskManager: plugins.taskManager }).catch(() => {});
```

Comment:
ideally this should only need to be started if user adds SLOs for the first time.

Reply:
Or in the case they have some existing SLOs and are only migrating from a previous version? Handling every cases might become tricky.
There is a feature flag that can be used to turn off this task, if someone's complaining about this task and don't use SLOs, it can be disabled easily.

Reply:
how can it be disabled in cloud? cloud has an allow list for which kibana configs can be added.

Reply:
Then the question is how can we update that allow list?

Reply:
i am not sure myself, on cloud first region you can add all configs but not on other regions.  but i think it's highly unlikely it will be needed in this case.